Sources/SwiftParser/Patterns.swift

```diff
@@ -53,6 +53,8 @@ extension Parser {
       case dollarIdentifier  // For recovery
       case letKeyword
       case varKeyword
+      case inoutKeyword
+      case borrowKeyword
```

Comment:
Same here: Could you name these the same as at the keyword and sort the entries in the `switch`es in the same order as the case declarations?